### PR TITLE
Second try fixing issue #8, because vscode tried to reformat the code…

### LINKED ISSUE #8: Window opens with incorrect size on dual screen setups
confirmed in win10 by PythonCode

### PR DESCRIPTION
**Pull request by pdamianik**
… in a wrong way
The issue #8 is fixed by getting the screen size from the screen modes instead of the screen itselft.

### CODE REVIEW

File: CONSTANTS.py [hunk 1]
```diff
@@ -72,8 +72,10 @@ def dumps(cls):
 
 DISPLAY=pyglet.canvas.get_display()#It took ages to find these functions, so don't question them.
 SCREEN=DISPLAY.get_default_screen()
-WIDTH=SCREEN.width
-HEIGHT=SCREEN.height
+SCREEN_MODES=SCREEN.get_modes()
+SCREEN_MODE=SCREEN_MODES[-1]
```
Comment (RiedleroD) [inline CONSTANTS.py]: on my laptop, the first one is actually the correct one and the last one the smallest one. If this is reversed on windows, I suggest going through all of them and getting the biggest by total pixels.
```
SCREEN_MODES.sort(reversed=True,key=lambda mode:mode.width*mode.height)
SCREEN_MODE=SCREEN_MODES[0]
```

**Review by RiedleroD — CHANGES_REQUESTED** [review]
see comments

File: CONSTANTS.py (hunk 1 above)
Comment (pdamianik) [reply CONSTANTS.py]: Changed with the next commit

**Review by RiedleroD — COMMENTED** [review]
I already merged this, but the review wasn't finished.
there you go.

**Comment by RiedleroD** [comment]
no I didn't request changes, wtf is github on today?

**Pull request merged**